test/test.yaml

```diff
@@ -1,14 +1,11 @@
-# This is the default Odyssey configuration file for the Prometheus MSLA. It is
-# meant to be paired with the latest Prometheus MSLA klipper config.
+# This is the Odyssey configuration file for local testing only. It is not intended
+# for real hardware, and does not carry particularly realistic values.
 
 # This section holds the config fields related to the printer, such as its serial
 # connection and its frame buffer specs
 printer:
-  serial: /dev/pts/12 # Mock port opened with `python3 test/mockSerial.py`
```

Comment:
Maybe we can come up with a way to automatically determine this by scanning for a mockSerial.py process?

Reply:
Yeaaah, this whole thing is getting a drastic revamp with the testing overhaul. This functionality will not be sticking around.